phpunit.xml

```diff
@@ -12,7 +12,7 @@
     </filter>
 
     <logging>
-        <log type="coverage-html" target="build/cov/html/" title="Str" charset="UTF-8" />
```

Comment:
Remove these attribute settings because they're deprecated on `phpunit 7.x` version.

The deprecated warning message is as follows:

```
Line 15:
  - Element 'log', attribute 'title': The attribute 'title' is not allowed.
  - Element 'log', attribute 'charset': The attribute 'charset' is not allowed.
```